File: test_4.py
class Calc:

  def func_4(len):

    tab = ''
    for i in range(1, len+1):
      tab += f'\t{i}'
    for j in range(1, len+1):
      tab += f'\n{j}\t'
      for i in range(1, len+1):
        tab += f'{i*j}\t'

    return tab

print(Calc.func_4(len = int(input())))

File: test_4.js
const readline = require(`readline`).createInterface( {
    input: process.stdin,
    output: process.stdout,
  });

class Calc {

    static func_4(len) {

    let tab = ``;
    for (let i=1; i<=len; i++) {
      tab += `\t${i}`
    }
    for (let j=1; j<=len; j++) {
      tab += `\n${j}\t`
      for (let i=1; i<=len; i++) {
        tab += `${i*j}\t`
      }
    }    
    return tab;
    }
}

readline.question("Введите число: ", function(input) {
    let len = parseInt(input);
    console.log(Calc.func_4(len));
    readline.close();
})